File: fc-chat/demo/part3/ch3/02-6-message-system-add-connection/src/main/java/net/prostars/messagesystem/dto/websocket/outbound/FetchConnectionsResponse.java
package net.prostars.messagesystem.dto.websocket.outbound;

import java.util.List;
import net.prostars.messagesystem.constant.MessageType;
import net.prostars.messagesystem.dto.domain.Connection;

public class FetchConnectionsResponse extends BaseMessage {

  private final List<Connection> connections;

  public FetchConnectionsResponse(List<Connection> connections) {
    super(MessageType.FETCH_CONNECTIONS_RESPONSE);
    this.connections = connections;
  }

  public List<Connection> getConnections() {
    return connections;
  }
}

File: fc-chat/demo/part3/with-push/src/main/kotlin/com/yscorp/withpush/messagesystem/dto/websocket/outbound/FetchConnectionsResponse.kt
package com.yscorp.withpush.messagesystem.dto.websocket.outbound

import com.yscorp.withpush.messagesystem.constant.MessageType
import com.yscorp.withpush.messagesystem.dto.domain.Connection

class FetchConnectionsResponse(val connections: List<Connection>) : BaseMessage(MessageType.FETCH_CONNECTIONS_RESPONSE) {
}
